docs/library/jesd204/versal_transceiver_subsystem/index.rst

```diff
@@ -0,0 +1,193 @@
+.. _versal_transceiver_subsystem:
+
+Versal Adaptive SoC Transceiver Subsystem
+=========================================
+
+Overview
+********
+
+The Versal Transceiver Subsystem can be used to configure the GTY and GTYP for Versal ACAP devices. In general,
+across all reference designs, the gigabit transceivers are configured specifically to the line rate of the HDL build.
```

Comment:
```suggestion
across all reference designs, the gigabit transceivers are configured specifically to the lane rate of the HDL build.
```
Definitely not wrong to use "line", but we tried to use mostly "lane" in the rest of the documentation (of course that there are still places where it's not, but we tend towards this)